.github/workflows/atomfeed_console_build_publish.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build and Publish atomfeed-console
+on:
+  repository_dispatch:
+    types: [atomfeed-console-trigger]
+  push:
+    branches: [master]
+    paths:
+      - 'atomfeed-console/**'
+      - '.github/workflows/atomfeed_console_build_publish.yml'
+jobs:
+  build-atomfeed-console:
+    name: Build atomfeed-console
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout atomfeed-console repo
+        uses: actions/checkout@v3
+        with:
+          repository: ICT4H/atomfeed-console
+      - name: Setup Java 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: "8"
+      # - name: Run Unit Tests
+      #   run: mvn --no-transfer-progress clean test
+      - name: Install bower
+        run: npm install -g bower 
+      - name: Package
+        run: mvn --no-transfer-progress clean package -DskipTests
```

Comment:
Why do we want to build the atomfeed console source code here ? Can we move this step to the atomfeed-console repository itself and consume only the bundled jar file here ?

Reply:
I also feel the same way. Should treat atomfeed-console as a 3rd party product and consume its jar/images directly as much as possible.

Reply:
yes. absolutely. atomfeed-console should be build as part of its own pipeline/GHA. and here we ought to just use the artefact and create/publish the docker image